### ISSUE TITLE
Potential Future Changes

### ISSUE THREAD

**Issue by C-Stew** [C-Stew/SkyrimSE-Modlist#4]
- Add images for greater clarity/flow
- Modulate the guide to be more than just one long page.
- Include a section for ENBless replacements.

- ~~Convert to HTML?~~ *Probably not until I learn actual web design lol*

*Feel free to suggest changes of your own in a new issue!*
